salalib/shapemap.h

```diff
@@ -222,7 +222,7 @@ class ShapeMap : public PixelBase
    mutable bool m_newshape;   // if a new shape has been added
    //
    // quick grab for shapes
-   pqvector<ShapeRef> **m_pixel_shapes;    // i rows of j columns
+   std::vector<std::vector<ShapeRef> > m_pixel_shapes;    // i rows of j columns

```

Comment:
I am so glad you tackled this one. This double pointer to a pqvector always made me feel queasy!